udata-tabular-preview/data-structure.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="bg-alt-green-tilleul-verveine fr-p-3v fr-my-2w" v-if="hasError">
+    <p class="fr-grid-row fr-m-0">
+      <span class="fr-icon-warning-line" aria-hidden="true"></span>
+      {{ $t("The data structure of this file failed to load.") }}
```

Comment:
The alignment of the error is not the same as in the Preview tab.

Reply:
I changed the preview error alignment to match this one